src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1616,10 +1301,10 @@ public CitizenData getCitizen(final int citizenId)
     /**
      * Checks if the achievements are valid.
      */
-    private void checkAchievements()
+    public void checkAchievements()
     {
         // the colonies size
-        final int size = this.citizens.size();
+        final int size = this.citizenManager.getCitizens().size();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "size". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "size" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1481)